src/__acceptance_tests__/__ssr_pages__/main-navigation-bar-large.tsx

```diff
@@ -23,7 +23,7 @@ const NavigationBarTest = (): JSX.Element => (
                     </Badge>
                 </NavigationBarAction>
                 <NavigationBarAction onPress={() => {}} aria-label="Open profile">
-                    <Avatar size={24} initials="ML" src="https://source.unsplash.com/600x600/?face" />
```

Comment:
unrelated change, fixed this test because it was using an external image and it was failing to load